phase2/Kconfig

```diff
@@ -39,6 +39,12 @@ config phase2.kubeadm.version
 
 	  Valid options are "stable" or a Google Cloud Storage link to a build.
 
+config phase2.kube_context_name
+    string "kube context name"
+    default ""
+    help
+	  The name of the context in kubeconfig.
```

Comment:
I would expand this description to indicate that the field is completely optional, and describe what will happen in the event that the user sets it.